addon/models/user.js

```diff
@@ -1,8 +1,8 @@
 import DS from 'ember-data';
 
-import OsfModel from 'ember-osf/models/base';
+import OsfModel from '../mixins/osf-model';
```

Comment:
No strong motivation; it seemed that some other files were starting to move this way and I was just matching styles.

It's entirely possible that there's a subtlety of addons where absolute imports would make more sense. If that's the case, I'm more than happy to change it back.